.tests/testScript_subsasgn.m

```diff
@@ -69,7 +69,7 @@
 
 %% expected to fail: assign dim to normal (doesn't call overloaded method)
 v1 = 1:4;
-shoulderror("v1([2,6]) = 9*u.lb;")
+shoulderror("v1([2,6]) = 9*u.lb;")%FIXME: something is wrong over here - no error has been thrown and this statement works well for me
```

Comment:
This still works fine for me.
@sky-s :should it fail?

Reply:
@sky-s need your input over here

Reply:
Added some clarifying comments